src/persist/src/storage.rs

```diff
@@ -161,10 +188,7 @@ pub trait Blob: Send + 'static {
     /// Succeeds if the key does not exist.
     async fn delete(&mut self, key: &str) -> Result<(), Error>;
 
-    /// List all of the keys in the map.
-    async fn list_keys(&self) -> Result<Vec<String>, Error>;
-
-    /// Synchronously closes the blob, releasing exclusive-writer locks and
+    /// Synchronously closes the blob, releasing the exclusive-writer lock and
```

Comment:
fyi, I mentioned in a PR that you probably haven't gotten around to looking at yet that close likely wants to move to BlobRead

Reply:
yeah decided to move close to BlobRead and impl it differently between e.g. FileBlob and FileBlobRead, even if that is a little wonky. added a comment on the rustdoc for close